### PR TITLE
Add course Ms SQL Server – Podstawy

### PR DESCRIPTION
**Pull request by DanaCyb**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ x] Include author(s) and platform where appropriate.
- [x ] Put lists in alphabetical order, correct spacing.
- [x ] Add needed indications (PDF, access notes, under construction).
- [x ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-pl.md [hunk 1]
```diff
@@ -86,6 +87,7 @@
 
 * [Kurs MySQL](https://www.youtube.com/playlist?list=PL748D0ACBEC371708)
 * [Kurs MySQL. Bazy danych, język zapytań SQL](https://www.youtube.com/playlist?list=PLOYHgt8dIdoymv-Wzvs8M-OsKFD31VTVZ) - Mirosław Zelent, Damian Stelmach
+* [Ms SQL Server – Podstawy](https://kursy.online-project.pl/product/ms-sql-server-podstawy) - Andrzej Śmigielski
```
Comment (eshellman) [inline courses/free-courses-pl.md:90]: bad link?

Comment (DanaCyb) [reply courses/free-courses-pl.md:90]: yes, sorry😢

Comment (eshellman) [reply courses/free-courses-pl.md:90]: I've looked at this again. link works, but the whole ordering thing with price zero makes it hard to confirm its free. Have you taken the course?

Comment (eshellman) [reply courses/free-courses-pl.md:90]: sorry for delays #9109

**Pull request closed without merging**